security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,26 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-52"]
+== OpenShift Compliance Operator 0.1.52
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.52:
```

Comment:
Maybe consider adding an attribute at some point? Not here/now, but in the future

Reply:
Solid idea - I'll poke around other releases and see what they're up to.